core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java

```diff
@@ -464,7 +465,15 @@ public void invoke() {
     /**
      * Executable name for Docker Compose.
      */
-    private static final String COMPOSE_EXECUTABLE = "docker-compose";
```

Comment:
Hi @bsideup .
I was trying to stay consistent with this (although I missed that it was still final) : 
https://github.com/testcontainers/testcontainers-java/blob/32b655190ddc7d2b24d136370cf4ca562a536903/core/src/main/java/org/testcontainers/utility/DockerMachineClient.java#L24-L31

I agree that your version is cleaner. I'll update the PR with that change.